components/Projects.js

```diff
@@ -9,41 +9,39 @@ export default function Projects() {
           Projects
         </h1>
       </div>
-      {/* Grid starts here */}
-      <div className="bg-[#F1F1F1] dark:bg-gray-900">
-        <div className="max-w-6xl mx-auto grid grid-cols-1 md:grid-cols-2 gap-8 py-20 pb-40">
-          {userData.projects.map((proj, idx) => (
-            <ProjectCard
-              title={proj.title}
-              link={proj.link}
-              imgUrl={proj.imgUrl}
-              number={`${idx + 1}`}
-            />
-          ))}
-        </div>
+      <div className="bg-[#F1F1F1] dark:bg-gray-900 p-10 grid grid-cols-1 md:grid-cols-2">
+        {userData.projects.map((project, index) => (
+          <ProjectCard projectID={index} />
+        ))}
       </div>
     </section>
   );
 }
 
-const ProjectCard = ({ title, link, imgUrl, number }) => {
+const ProjectCard = ({ projectID }) => {
+  const project = userData.projects[projectID];
+
   return (
-    <a href={link} className="w-full block shadow-2xl">
-      <div className="relative overflow-hidden">
-        <div className="h-72 object-cover">
-          <img
-            src={imgUrl}
-            alt="portfolio"
-            className="transform hover:scale-125 transition duration-2000 ease-out object-cover h-full w-full"
-          />
-        </div>
-        <h1 className="absolute top-10 left-10 text-gray-50 font-bold text-xl bg-red-500 rounded-md px-2">
-          {title}
-        </h1>
-        <h1 className="absolute bottom-10 left-10 text-gray-50 font-bold text-xl">
-          {number.length === 1 ? "0" + number : number}
-        </h1>
+    <div className="rounded flex flex-col justify-between transform hover:scale-105 transition duration-200 shadow-md my-4 mx-4">
+      <img className="w-full shadow-sm" src={project.imgUrl}></img>
+      <div className="px-6 py-4 justify-self-start">
```

Comment:
I'm trying to get the text to stick to the bottom of the image. From the tailwind docs it seems like `justify-self-start` should accomplish this, but it doesn't seem to be working. Hoping you may have some insight here!

Reply:
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/6580472/155042974-8dd0b200-a8f6-4e30-83e3-284d6f44a449.png">

Reply:
Yep - I think it has to do with the images being different heights.

I'd suggest you overlay the title / description block over the _top_ of the image, though. See my draft PR for implementaiton.